sections/featured-product.liquid

```diff
@@ -0,0 +1,615 @@
+                    <span class="visually-hidden">{{ 'products.product.quantity.increase' | t: product: product.title | escape }}</span>
+                    {% render 'icon-plus' %}
+                  </button>
+                </quantity-input>
+              </div>
+            {%- when 'popup' -%}
+                <modal-opener class="product-popup-modal__opener no-js-hidden" data-modal="#PopupModal-{{ block.id }}" {{ block.shopify_attributes }}>
+                  <button id="ProductPopup-{{ block.id }}" class="product-popup-modal__button link" type="button" aria-haspopup="dialog">{{ block.settings.link_label | default: block.settings.page.title }}</button>
+                </modal-opener>
+                <a href="{{ block.settings.page.url }}" class="product-popup-modal__button link no-js">{{ block.settings.link_label }}</a>
+            {%- when 'share' -%}
+              <share-button class="share-button" {{ block.shopify_attributes }}>
+                <button class="share-button__button button button--tertiary hidden">
+                  {% render 'icon-share' %}
+                  {{ 'general.share.share' | t }}
+                </button>
+                <details>
+                  <summary class="share-button__button button button--tertiary">
+                    {% render 'icon-share' %}
+                    {{ 'general.share.share' | t }}
+                  </summary>
+                  <div class="share-button__fallback">
+                    <div class="field">
+                      <input type="text"
+                            class="field__input"
+                            id="url"
+                            value="{{ shop.url | append: product.url }}"
+                            placeholder="{{ 'general.share.share_url' | t }}"
+                            onclick="this.select();"
+                            readonly
+                      >
+                      <label class="field__label" for="url">{{ 'general.share.share_url' | t }}</label>
+                    </div>
+                    <button class="button button--tertiary">
+                      {% render 'icon-clipboard' %}
+                      {{ 'general.share.copy_to_clipboard' | t }}
+                    </button>
+                    <span id="ShareMessage-{{ section.id }}" class="share-button__message hidden" role="status" aria-hidden="true">
+                      {{ 'general.share.success_message' | t }}
+                    </span>
+                  </div>
+                </details>
+              </share-button>
+              <script src="{{ 'share.js' | asset_url }}" defer="defer"></script>
+            {%- when 'variant_picker' -%}
+              {%- unless product.has_only_default_variant -%}
+                {%- if block.settings.picker_type == 'button' -%}
+                  <variant-radios class="no-js-hidden" data-section="{{ section.id }}" data-url="{{ product.url }}" data-update-url="false" {{ block.shopify_attributes }}>
```

Comment:
out of curiosity, are hover styles for radio button going to be tackled here ? I assume it's going to be its own PR. ([figma reference](https://screenshot.click/21-08-k34r2-4gn0p.png))

Reply:
I'll need to look at this closely, the only piece I believe we were missing is the Sold out UI that would come in later.

Reply:
I honestly had no idea these styles existed. They're probably new 👀 

We can do it on the sold out UI PR one like meli said.